nuxeo-distribution/nuxeo-distribution-resources/src/main/resources/templates-tomcat/common-base/nuxeo.defaults

```diff
@@ -15,6 +15,10 @@ nuxeo.db.embeddedDatasources=true
 nuxeo.db.xaMode=false
 nuxeo.db.commonds=jdbc/nuxeo
 
+nuxeo.db.blocking-timeout-millis=10000
+nuxeo.db.idle-timeout-minutes=0
+nuxeo.db.min-pool-size=5
+
```

Comment:
For what I have understood, the chain of "nuxeo.defaults" files overrides at will the different values. There is a ticket for a global clean up of all of them (plus some hard-coded values in some templates), because values are declared multiple times in multiple places.
We agreed to put it as close as possible to the corresponding template, instead of far away from it.
This is how it has already been validated and merged in 6.0.